pkg/networkmanager/network-interface.jsx

```diff
@@ -192,6 +194,8 @@ export const NetworkInterfacePage = ({
                 desc = _("VLAN");
             else if (cs.type == "bridge")
                 desc = _("Bridge");
+            else if (cs.type == "wireguard")
+                desc = "WireGuard";
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17653-20220824-154453-fde0a772-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)